docs/content/kor/search/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+date: 2022-3-31
+title: Search
+draft: false

```

Comment:
What is this file used for?

Reply:
Just the first commit is going to be merged, the second one was added so you can test the feature :D This is mentioned in the description of the PR :D